app/coins/btc.js

```diff
@@ -53,9 +53,9 @@ module.exports = {
 	name:"Bitcoin",
 	ticker:"BTC",
 	logoUrlsByNetwork:{
-		"main":"/img/logo/btc.svg",
-		"test":"/img/logo/tbtc.svg",
-		"regtest":"/img/logo/tbtc.svg"
+		"main":"./img/logo/btc.svg",
```

Comment:
FYI this will work the same without the leading `./` (here and everywhere else in this PR)

Reply:
Which way is more idiomatic? I like the explicitness of `./`, but if it's better without it, then I can remove it.

Reply:
I think that without is more idiomatic, but no big deal either way. If you like the explicitness then go for it :)